packages/enzyme-matchers/src/utils/html.js

```diff
@@ -39,19 +39,22 @@ function mapWrappersHTML(wrapper: EnzymeObject): Array<string> {
   });
 }
 
-export default function printHTMLForWrapper(wrapper: EnzymeObject): string {
+export default function printHTMLForWrapper(
+  wrapper: EnzymeObject,
+  hostNodesOnly?: Boolean
+): string {
   switch (wrapper.getElements().length) {
     case 0: {
       return '[empty set]';
     }
     case 1: {
       if (isShallowWrapper(wrapper)) {
-        // This is used to clean up in any awkward spacing in the debug output.
-        // <div>  <Foo /></div> => <div><Foo /></div>
-        return wrapper.debug().replace(/\n(\s*)/g, '');
```

Comment:
I probably should have mentioned the removal of the white-space stripping in the PR description. The awkward spacing mentioned in the above comment was addressed in a v1 release, I believe. I’ll try to track it down in the change log again.

Reply:
Found it! It was addressed in [the 2.9 release](https://github.com/airbnb/enzyme/blob/master/CHANGELOG.md#290).